src/test/java/parameters/TestData.java

```diff
@@ -0,0 +1,16 @@
+package parameters;
+
+import org.testng.annotations.DataProvider;
+
+import static constants.TestConstants.*;
+
+public class TestData {
+
+    @DataProvider
+    public static Object[][] yandexMailCredentials() {
```

Comment:
Usualy there is no need to keep datatprovider in a separate class - you can keep it in the same class with test that uses it.

Reply:
Done!